image/README.md

```diff
@@ -1,14 +1,42 @@
-# Imaging
```

Comment:
Your algorithm needs to be in its own folder and not in the image parent directory. Please move your files into its own project directory.